routes/sites.js

```diff
@@ -59,22 +59,27 @@ router.get('/', async function(req, res, next) {
     // Variables to track pagination of user's repos in case user has more than 100
     let pageCount = 1
     let hasNextPage = true;
-    const filePath = `https://api.github.com/user/repos?per_page=100&page=`;
+    const endpoint = `https://api.github.com/orgs/${ISOMER_GITHUB_ORG_NAME}/repos`;
 
     // Loop through all user repos
     while (hasNextPage) {
-      const resp = await axios.get(filePath + pageCount, {
+      const resp = await axios.get(endpoint, {
+        params: {
+          per_page: 100,
+          page: pageCount,
+          sort: "full_name",
+        },
         headers: {
           Authorization: `token ${access_token}`,
-          "Content-Type": "application/json"
+          Accept: "application/vnd.github.baptiste-preview+json",
         }
       })
 
       // Filter for isomer repos
       const isomerRepos = resp.data.reduce((acc, repo) => {
-        const { updated_at, full_name } = repo
+        const { permissions, updated_at, full_name } = repo
         const fullName = full_name.split('/')
```

Comment:
Remove `full_name`.

We can extract `name` from the repo API response and use it. `name` is the repo name without the org name

Reply:
done